src/components/nav-item/styles.ts

```diff
@@ -0,0 +1,51 @@
+import { styled } from '@linaria/react'
+
+export const ElNavItemLabelContainer = styled.span`
+  display: flex;
+  padding: var(--spacing-spacing-half, 2px);
+  align-items: flex-start;
+
+  font-family: var(--font-family, Inter);
+  font-size: var(--font-size-sm, 14px);
+  font-style: normal;
+  font-weight: 500;
+  line-height: var(--line-height-sm, 20px);
+  letter-spacing: var(--letter-spacing-sm, -0.14px);
+`
+
```

Comment:
**thought:** I didn't realise this with the `NavDropdownButton` work, but that component is really just a `NavItem` with an icon and a menu, so I wonder if we should have `NavDropdownButton` use `NavItem` (or `ElNavItemButton`) under the hood 🤔 

Not something we need to do right now, just flagging that I think we missed this relationship when planning out the sequence of the work (i.e. NavItem should have come first, then NavDropdownButton).

Reply:
ah yes, both of them have different variant (i.e dropdown-button doesn't have `selected` variant) I think that is the reason, but actually it could be combined.

Reply:
I can make another pr for that later

Reply:
I believe the dropdown button would show as active when it's open and/or when one of it's items is active; but yes, we'd need to confirm behaviour with design before doing that work. I don't think it's costly to have this duplication though, so I think DRYing it up would be pretty low priority.